lib/pg_saurus/connection_adapters/postgresql_adapter/foreign_key_methods.rb

```diff
@@ -44,14 +44,14 @@ def add_foreign_key(from_table, to_table, options = {})
     # See activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb
     #
     # Pass in the option remove_index: true to remove index as well.
-    def remove_foreign_key(from_table, options_or_to_table = {})
-      if options_or_to_table.is_a?(Hash) && options_or_to_table[:remove_index]
-        column = options_or_to_table[:column]
+    def remove_foreign_key(from_table, to_table = nil, **options)
```

Comment:
Corresponding method in:
- Rails 6.0.0: [ActiveRecord::ConnectionAdapters::SchemaStatements#remove_foreign_key](https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#1026)
- Rails 6.1.0: [ActiveRecord::ConnectionAdapters::SchemaStatements#remove_foreign_key](https://github.com/rails/rails/blob/v6.1.0/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#1094)

The main update here is to refactor the generic positional argument `options_or_to_table` and make the method signature more consistent with the super method.